src/Symfony/Bundle/WebProfilerBundle/Controller/RouterController.php

```diff
@@ -99,4 +103,26 @@ private function getTraces(RequestDataCollector $request, $method)
 
         return $matcher->getTracesForRequest($traceRequest);
     }
+
+    /**
+     * Replaces scalars wrapped as Data instances by their actual values and removes other values.
```

Comment:
In fact, we'd better serialize the original request in the RequestDataCollector, don't you thing?